metric-anomaly-detector/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+plugins {
+    java
+    application
+    jacoco
+    id("org.hypertrace.jacoco-report-plugin")
+}
+
+application {
+    mainClass.set("org.hypertrace.alerting.metric.anomaly.detector.MetricAnomalyDetector")
+}
+
+dependencies {
+    implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.23")
+    implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.23")
+
+    // Required for the GRPC clients.
+    runtimeOnly("io.grpc:grpc-netty:1.36.1")
+
+    implementation("com.typesafe:config:1.4.1")
+    implementation("io.confluent:kafka-avro-serializer:6.0.1")
```

Comment:
Pl. keep only used one. Whenever you start using it, along with it, lets add deps.